src/scripts/OSFramework/OSUI/Pattern/Progress/Bar/ProgressBar.ts

```diff
@@ -95,7 +95,8 @@ namespace OSFramework.OSUI.Patterns.Progress.Bar {
 		 * @memberof OSFramework.Patterns.Progress.Bar.Bar
 		 */
 		protected setA11YProperties(): void {
-			Helper.A11Y.AriaLabelledBy(this.selfElement, this.contentElemId);
+			if (this.contentElem.innerHTML) Helper.A11Y.AriaLabelledBy(this.selfElement, this.contentElem.id);
+			else Helper.A11Y.AriaLabel(this.selfElement, ProgressEnum.AriaLabel.Progress);
```

Comment:
Have mix feelings about this Progress enum due to translations, but this is something that can be changed through extensibility...